templates/internet-of-things/index.html

```diff
@@ -11,7 +11,8 @@
   <div class="row u-vertically-center">
     <div class="col-6">
       <h1>Ubuntu for the Internet&nbsp;of&nbsp;Things</h1>
-      <p>From home control to drones, robots and industrial systems, Ubuntu is the <a class="p-link--inverted" href="/embedded">new standard for embedded Linux</a>. Whether you choose the classic Ubuntu Server or the new Ubuntu Core you get robust security, app stores and reliable updates.</p>
+      <p>From smart homes to smart drones, robots, and industrial systems, Ubuntu is the <a class="p-link--inverted" href="/embedded" style="font-style: italic;">new standard for embedded Linux</a>. Get the world’s best security, a custom app store, a huge developer community and reliable updates.
```

Comment:
I don't think we want to add italic links as we don't anywhere else.

Reply:
This was @anasereijo suggestion as the link was not visible. @anasereijo can you please propose it as a vanilla pattern and/or discuss alternatives with vanilla team.